gammapy/modeling/models/tests/test_spectral.py

```diff
@@ -589,8 +589,21 @@ def test_ecpl_model(self):
         out = model.evaluate_error(0.1 * u.TeV)
         assert_allclose(out.data, [1.548176e-10, 1.933612e-11], rtol=1e-3)
 
-    def test_naima_model_error_proprgation(self):
-        # Regression test for Naima model
-        # https://github.com/gammapy/gammapy/issues/2190
-        # TODO: implement test case. Move to Naima model tests!
-        pass
+
+@requires_dependency("naima")
+def test_naima_model_error_propagation():
```

Comment:
Suggest to move the `evaluate_error` call the the Naima example we already have, as here:
https://github.com/gammapy/gammapy/pull/2602/files
Advantage: less test code.

Reply:
Done